tests/system/plugin-jar.test.ts

```diff
@@ -73,3 +71,45 @@ test('inspect on user created jar (same as altered)', async (t) => {
     );
   }
 });
+
+test('inspect on directory with jars', async (t) => {
+  const result = await plugin.inspect(
```

Comment:
we said this will only be triggered behind  cli option like '--all-jars', I am not seeing any option handling anywhere

Reply:
yep need to speak with Or to agree what the option will be, i'll assume '--all-jars' and push update